### PR TITLE
updating UBC function & tests based on latest changes

### PR DESCRIPTION
**Pull request by vermashivam679**

### CODE REVIEW

File: hueniversitypy/theme_ubc.py [hunk 1]
```diff
@@ -1,21 +1,58 @@
...
     >>> alt.themes.register('theme_ubc', theme_ubc)
     >>> alt.themes.enable('theme_ubc')
-
     """
+    #attribution to Sergio Sanchez 
+    #https://towardsdatascience.com/consistently-beautiful-visualizations-with-altair-themes-c7f9f889602
+
+    font = "Arial"
+    labelFont = "Arial" 
+    sourceFont = "Arial"
```
Comment (reikookamoto) [inline hueniversitypy/theme_ubc.py]:
```suggestion
```
Don't think `SourceFont` is used here

File: hueniversitypy/theme_ubc.py [hunk 2]
```diff
@@ -1,21 +1,58 @@
...
+    # Axes
+    axisColor = "#000000"
+    gridColor = "#DEDDDD"
```
Comment (reikookamoto) [inline hueniversitypy/theme_ubc.py]:
```suggestion
```
Don't think these variables are needed in the code

File: hueniversitypy/theme_ubc.py [hunk 3]
```diff
@@ -1,21 +1,58 @@
...
+    # Colors
+    main_palette = ['#002145', '#0055B7', '#00A7E1', '#40B4E5', '#6EC4E8', '#97D4E9']
```
Comment (reikookamoto) [inline hueniversitypy/theme_ubc.py]:
```suggestion
    ubc_palette = ['#002145', '#0055B7', '#00A7E1', '#40B4E5', '#6EC4E8', '#97D4E9']
```
Just changed palette name so it's clear which one we're using.

Comment (vermashivam679) [reply hueniversitypy/theme_ubc.py]: If I change the name here I will have to change below as well. Please accept this PR I will modify it in the next edit along with renaming `test_plot` to `ubc_plot`

Comment (reikookamoto) [reply hueniversitypy/theme_ubc.py]: Ah good point.

**Pull request merged**